data/mapping/mapping.json

```diff
@@ -69,20 +69,7 @@
           }
         },
         "location": {
```

Comment:
Rather than reshaping the data could `location` be changed to `location.coordinates`? According to the [geo-point docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html#CO181-4) this format is supported.